shared/common-adapters/icon.native.js

```diff
@@ -32,13 +32,30 @@ class Icon extends Component<void, Exact<Props>, void> {
     const textAlign = this.props.style && this.props.style.textAlign
     const backgroundColor = this.props.style && {backgroundColor: this.props.style.backgroundColor} || {}
 
-    // Color is for our fontIcon and not the container
-    let containerProps = {...this.props.style}
-    delete containerProps.color
-    delete containerProps.width
-    delete containerProps.height
-    delete containerProps.textAlign
-    delete containerProps.fontSize
+    // Extract style props for our container and box
+    let containerProps = {
+      ...this.props.style,
+      bottom: undefined,
+      color: undefined,
+      fontSize: undefined,
+      height: undefined,
+      left: undefined,
+      position: undefined,
+      right: undefined,
+      textAlign: undefined,
+      top: undefined,
+      width: undefined,
+    }
+
+    const clickableBoxStyle = {
+      bottom: this.props.style && this.props.style.bottom,
+      height: this.props.style && this.props.style.height,
+      left: this.props.style && this.props.style.left,
+      position: this.props.style && this.props.style.position,
+      right: this.props.style && this.props.style.right,
+      top: this.props.style && this.props.style.top,
+      width: this.props.style && this.props.style.width,
+    }
```

Comment:
would `const {bottom, height, left, ...etc} = this.props.style || {}` work here?